src/components/BottomBar/BottomBarComponent.js

```diff
@@ -33,6 +37,51 @@ const StyledLabel = styled.div`
   text-align: center;
 `;
 
+const UploadFileButton = () => {
+  const classes = useStyles();
+  const dispatch = useDispatch();
+
+  const uploadFolderId = useSelector(NotesMindMapSelectors.getSelectedNoteId);
+  const fileInputRef = React.createRef();
+
+  function onUploadButtonClick() {
+    fileInputRef.current.click();
+  }
+
+  function handleSelectedFiles(event) {
+    if (event.target.files.length > 0) {
+      const files = Array.from(event.target.files);
+
+      files.forEach(item => {
+        item.uploadFolderId = uploadFolderId;
+        item.abortController = new window.AbortController();
+      });
+
+      dispatch(UploadsActions.list.addedFiles(files));
+      fileInputRef.current.value = null;
+    }
+  }
```

Comment:
What is this code for? Did you copy it from somewhere? How are uploads related to note details?

Reply:
It's bottom bar functionality. It's not related to note page. 

It's good workflow to create branch and then regularly merge to master from one branch. So we if we have **note-page** branch:
- I add note page and then PR
- then you checkout to this branch and run app, check if it's working well
- then you review file changes and search for typos and mistakes
- if found comment on PR
- if not found merge and if some changes to be made related to this branch you comment continiously on merged branch 
- later I make changes and commit
- because it's same branch more merges can be made

Currently whats happening is that not any of my code goes to master, so the following problems occur

- rebase bugs. instead of rebase we could just merged and then fixed problem in later commits
- merge conflicts, everytime merge is happening conflicts are appearing, so it's good to merge often so the conflicts do not accumulate

Currently we can go forever with this branch. You will comment, I will make change and everytime master changes I will need resoleve more and more conflicts. So please merge and point out problems is new issues.

Reply:
My bad. Should have checked out to master and then made changes. When have merge conflicts please mention me I will resolve